src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -46,12 +46,15 @@ const Tabs = ({ children, id, modifyHash = true, testId, onToggle }: Props) => {
         'aria-labelledby': `${sectionId}-tab`,
         hidden: selectedTabIndex !== index,
         id: sectionId,
+        isSingleTab: filteredChildren.length === 1,
```

Comment:
In most usages of this, we've had to go with a negation of `!isSingleTab`. Perhaps it would be better to change this to a `isMultiTab` variable instead so that usages of it don't need the negation.